common/src/main/resources/startup.properties

```diff
@@ -332,3 +333,6 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 #*.falcon.graphite.port=2003
 #*.falcon.graphite.frequency=1
 #*.falcon.graphite.prefix=falcon
+
+# Property to remove postProcessing
+*.falcon.postprocessing.enable=true
```

Comment:
This doesn't need to be startup property. It can be a runtime property that will allow the admin to change it at runtime.